[forge:x64-windows] Compiles ok, but Forge_FOUND==false after find_package

vcpkg version  2021-01-13-d67989bce1043b98092ac45996a8230a059a2d7e
forge version 1.0.5
OS Windows 10

- `.\vcpkg.exe install forge:x64-windows` successful
- find_package(Forge CONFIG REQUIRED),    target_link_libraries(main PRIVATE Forge::forge) provided
- VisualStudio->Create New CMake project
- Insert `find_package(Forge CONFIG REQUIRED)` in generated CMakeLists.txt
- Error: 
[CMake] CMake Error at C:/Dev/vcpkg/scripts/buildsystems/vcpkg.cmake:861 (_find_package):
1> [CMake]   Found package configuration file:
1> [CMake] 
1> [CMake]     C:/Dev/vcpkg/installed/x64-windows/share/forge/ForgeConfig.cmake
1> [CMake] 
1> [CMake]   but it set Forge_FOUND to FALSE so package "Forge" is considered to be NOT
1> [CMake]   FOUND.
1> [CMake] Call Stack (most recent call first):
1> [CMake]   forgetest/CMakeLists.txt:6 (find_package)
1> [CMake] -- Configuring incomplete, errors occurred!

[CMakeOutput.log](https://github.com/microsoft/vcpkg/files/6345678/CMakeOutput.log)

--- a/ports/forge/CONTROL
+++ b/ports/forge/CONTROL
@@ -1,5 +1,0 @@
-Source: forge
-Version: 1.0.5
-Description: Helps with high performance visualizations involving OpenGL-CUDA/OpenCL interop.
-Build-Depends: glfw3, glm, glbinding, freetype, boost-functional, freeimage, fontconfig (!windows)
-Supports: x64

--- a/ports/forge/fix-usage.patch
+++ b/ports/forge/fix-usage.patch
@@ -1,0 +1,29 @@
+diff --git a/CMakeModules/ForgeConfig.cmake.in b/CMakeModules/ForgeConfig.cmake.in
+index e4c7737..6429069 100644
+--- a/CMakeModules/ForgeConfig.cmake.in
++++ b/CMakeModules/ForgeConfig.cmake.in
+@@ -52,16 +52,18 @@ set(Forge_FOUND OFF)
+ 
+ if (NOT TARGET Forge::forge AND
+         NOT TARGET forge AND
+-        EXISTS @PACKAGE_CMAKE_DIR@/ForgeTargets.cmake)
+-    include(@PACKAGE_CMAKE_DIR@/ForgeTargets.cmake)
++        EXISTS ${CMAKE_CURRENT_LIST_DIR}/ForgeTargets.cmake)
++    include(${CMAKE_CURRENT_LIST_DIR}/ForgeTargets.cmake)
+ endif ()
+ 
+ if (TARGET Forge::forge)
+-    get_property(config TARGET Forge::forge PROPERTY IMPORTED_CONFIGURATIONS)
+-    if(NOT config)
+-      set(config "NOCONFIG")
++    get_property(configs TARGET Forge::forge PROPERTY IMPORTED_CONFIGURATIONS)
++    if(NOT configs)
++      set(configs "NOCONFIG")
+     endif()
+-    get_property(loc TARGET Forge::forge PROPERTY IMPORTED_LOCATION_${config})
++    foreach(config ${configs})
++        get_property(loc TARGET Forge::forge PROPERTY IMPORTED_LOCATION_${config})
++    endforeach()
+ endif ()
+ 
+ if ((TARGET Forge::forge AND EXISTS ${loc})

--- a/ports/forge/portfile.cmake
+++ b/ports/forge/portfile.cmake
@@ -8,7 +8,9 @@ vcpkg_from_github(
     REF 1a0f0cb6371a8c8053ab5eb7cbe3039c95132389 # v1.0.5
     SHA512 8f8607421880a0f0013380eb5efb3a4f05331cd415d68c9cd84dd57eb727da1df6223fc6d65b106675d6aa09c3388359fab64443c31fadadf7641161be6b3b89
     HEAD_REF master
-	PATCHES fix-static_build.patch
+    PATCHES
+        fix-static_build.patch
+        fix-usage.patch
 )
 
 vcpkg_configure_cmake(

--- a/ports/forge/vcpkg.json
+++ b/ports/forge/vcpkg.json
@@ -1,0 +1,20 @@
+{
+  "name": "forge",
+  "version-semver": "1.0.5",
+  "port-version": 1,
+  "description": "Helps with high performance visualizations involving OpenGL-CUDA/OpenCL interop.",
+  "homepage": "https://github.com/arrayfire/forge",
+  "supports": "x64",
+  "dependencies": [
+    "boost-functional",
+    {
+      "name": "fontconfig",
+      "platform": "!windows"
+    },
+    "freeimage",
+    "freetype",
+    "glbinding",
+    "glfw3",
+    "glm"
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2046,7 +2046,7 @@
     },
     "forge": {
       "baseline": "1.0.5",
-      "port-version": 0
+      "port-version": 1
     },
     "fp16": {
       "baseline": "2021-02-21",

--- a/versions/f-/forge.json
+++ b/versions/f-/forge.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b160185b8e2d1f0bd2268cb56d73e0e6a2940f69",
+      "version-semver": "1.0.5",
+      "port-version": 1
+    },
+    {
       "git-tree": "29625ec4b111b2a3925f77222d9aa600229c8d96",
       "version-string": "1.0.5",
       "port-version": 0

